src/web/mjs/engine/DownloadManager.mjs

```diff
@@ -36,12 +37,53 @@ export default class DownloadManager extends EventTarget {
         }
     }
 
+    /**
+     * Returns true if the download queue is paused.
+     */
+    isQueuePaused() {
+        return this.queuePaused;
+    }
+
+    /**
+     * Pause the download queue processing.
+     */
+    pauseQueue() {
+        this.queuePaused = true;
+    }
+
+    /**
+     * Start the download queue processing if it was paused.
+     */
+    startQueue() {
+        this.queuePaused = false;
+        this.processQueue(); // Start processing the queue again
```

Comment:
This call seems redundant and may be removed as there is a worker running calling this every 250ms.